index.html

```diff
@@ -0,0 +1,772 @@
+      <nav class="naa-1">
+        <div class="nav-logo" style="background-color: black">
+          <a href="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152646mp_/https:/thenextweb.com/"><svg
+              xmlns="http://www.w3.org/2000/svg" viewBox="0 0 66 16">
+              <path fill="#FFF"
+                d="M34 16h-6v-5.2c0-.7-.3-1.3-.7-1.8L25 6.7c-.5-.4-1.1-.7-1.8-.7H22v10h-6V0h8.7c1.3 0 2.6.5 3.5 1.5l4.3 4.3c1 .9 1.5 2.2 1.5 3.5V16zM60 0v5.2c0 .7-.3 1.3-.7 1.8L57 9.3c-.5.5-1.1.7-1.8.7h-2.5c.8-.9 1.3-2.1 1.3-3.3V0h-6v5.2c0 .7-.3 1.3-.7 1.8L45 9.3c-.5.5-1.1.7-1.8.7H42V0h-6v16h8.7c1.3 0 2.6-.5 3.5-1.5l2.8-2.8V16h5.7c1.3 0 2.6-.5 3.5-1.5l4.3-4.3c1-.9 1.5-2.2 1.5-3.5V0h-6zM14 0H0v6h4v10h6V6h4" />
+            </svg></a>
+        </div>
+        <a href="#">
+          <div class="hambutt">
+            <div class="ham"></div>
+            <div class="ham"></div>
+            <div class="ham"></div>
+          </div>
+        </a>
+        <a href="#"><img class="srchicon" alt="search icon" src="Assets/Images/magnifying-glass.svg" /></a>
+      </nav>
+      <nav class="na-2">
+        <div class="nav-ul-l">
+          <ul class="nav-ul">
+            <li><a href="#">News</a></li>
+            <li><a href="#">Events</a></li>
+            <li><a href="#">Business</a></li>
+            <li><a href="#">AMAs</a></li>
+            <li><a href="#">Spaces</a></li>
+            <li><a href="#">Terms & Conditions</a></li>
+          </ul>
+        </div>
+        <div class="nav-ul-r">
+          <ul class="nav-ul">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Advertise</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="na-3">
+        <ul class="nav-ul">
+          <li><a href="#">LATEST</a></li>
+          <li><a href="#">HARD FORK</a></li>
+          <li><a href="#">PLUGGED</a></li>
+          <li><a href="#">README</a></li>
+          <li><a href="#">GROWTH QUARTERS</a></li>
+          <li><a href="#">SHIFT</a></li>
+          <li><a href="#">NEURAL</a></li>
+        </ul>
+        <img class="srchicon" alt="search icon" src="Assets/Images/magnifying-glass.svg" />
+      </nav>
+      <nav class="na-4">
+        <a href="#">Calling young digital talent in the Netherlands – Be recognized in
+          the T500 →</a>
+      </nav>
+    </nav>
+  </section>
+
+  <section class="main-sec">
+    <!--main tag-->
+    <section class="article3">
+      <article class="art-cont art-main">
+        <div class="art1 art123">
+          <a class="art-cover" href="#"></a>
+          <header class="arthead">
+            <div class="big-logo">
+              <img alt="" src="Assets/Images/big-logo.svg" />
+            </div>
+            <div class="date">Tuesday — January 26, 2021</div>
+            <div class="opal">
+              <p>
+                Opal cuts off your apps' internet access to help you stop
```

Comment:
This title should be enclosed in both `a` and `hx` tags. Do the same thing for all the titles in the Highlight section